### PR TITLE
feat: add default entrypoint for the container image

### PR DESCRIPTION
**Pull request by onur-ozkan**
I think most developers expect the application to start when running `docker run $image` instead of terminating the process without doing anything. This PR addresses that expectation.

### CODE REVIEW

**Comment by reecepbcups** [comment]
I would think developers would expect the relayer binary to run (i.e. help menu) rather than to start. `ENTRYPOINT ["rly"]` may be more appropriate. Thoughts @onur-ozkan ?

**Comment by onur-ozkan** [comment]
I am unsure how this will help when we are outside the container. `--help`(or any command that dumps the command interface) is mostly useful when we are actually using the CLI tool, but in most scenarios when pulling the image you just want to start the application in the container. This is how it's done in other cosmos projects as well like [ibc-go](https://github.com/cosmos/ibc-go/blob/6fa670af92378090a38838cdbf0331d07a25d0b5/Dockerfile#L35) and [gaia](https://github.com/cosmos/gaia/blob/main/Dockerfile#L20).

**Comment by onur-ozkan** [comment]
> I am unsure how this will help when we are outside the container. `--help` (or any command that dumps the command interface) is mostly useful when we are actually using the CLI tool, but in most scenarios you just want to start the application in the container. This is how it's done in other cosmos projects as well like [ibc-go](https://github.com/cosmos/ibc-go/blob/6fa670af92378090a38838cdbf0331d07a25d0b5/Dockerfile#L35) and [gaia](https://github.com/cosmos/gaia/blob/main/Dockerfile#L20).

**Comment by onur-ozkan** [comment]
> I am unsure how this will help when we are outside the container. `--help`(or any command that dumps the command interface) is mostly useful when we are actually using the CLI tool, but in most scenarios you just want to start the application in the container. This is how it's done in other cosmos projects as well like [ibc-go](https://github.com/cosmos/ibc-go/blob/6fa670af92378090a38838cdbf0331d07a25d0b5/Dockerfile#L35) and [gaia](https://github.com/cosmos/gaia/blob/main/Dockerfile#L20).

Oh, seems like ibc-go image does the exact thing you said above.

**Comment by reecepbcups** [comment]
@onur-ozkan Yea i think just `rly` is the best approach. Then adding on arguments as you need them (query, tx, start, etc).

I am going to leave this for someone else to decide on the best approach ~~as this is a breaking change~~. Thanks for the contribution here! Great thing for us to discuss & add ideally

**Comment by onur-ozkan** [comment]
> this is a breaking change.

Could you please clarify this a bit ? :slightly_smiling_face: At the moment there is no default entry point specified, so people have to specify one manually to make this container to do something useful. By doing so, the entry point we are adding with this PR will be overridden which means they won't notice any difference.

**Comment by reecepbcups** [comment]
Sorry, I was thinking docker CMD and not Entrypoint. Entrypoint is not a breaking change. Updated title and marketed out my comment. Thanks!

File: Dockerfile [hunk 1]
```diff
@@ -68,3 +68,5 @@ COPY --from=busybox-min --chown=100:1000 /home/relayer /home/relayer
 
 WORKDIR /home/relayer
 USER relayer
+
+ENTRYPOINT ["rly", "start"]
```
Comment (reecepbcups) [inline Dockerfile:72]: `ENTRYPOINT ["rly", "start"]` (like Gaia) or `ENTRYPOINT ["rly"]` (like IBC-Go)

**Review by reecepbcups — APPROVED** [review]
I am just going to take action here and approve it as is. thanks! sorry for the wait here

File: Dockerfile (hunk 1 above)
Comment (reecepbcups) [reply Dockerfile:72]: going with the gaia approach, does not matter since it can be overridden if the user decides too. default just starts

**Comment by reecepbcups** [comment]
Ci failure is due to rate limiting, will re-run this later

**Pull request merged**